regression-test/suites/nereids_syntax_p0/slow_fold_constant_case_when.groovy

```diff
@@ -0,0 +1,824 @@
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END )
+                END AS `_______0` , sum(c24) AS `c24` , sum(c28) AS `c28` , sum(c26) AS `c26` , sum(c21) AS `c21` , sum(c25) AS `c25` , sum(c29) AS `c29` , sum(c5) AS `c5` , sum(c6) AS `c6` , sum(c7) AS `c7` , sum(c15) AS `c15` , sum(c4) AS `c4` , sum(c27) AS `c27` , sum(c19) AS `c19` , sum(c18) AS `c18` , sum(c20) AS `c20` , sum(c8) AS `c8` , sum(c13) AS `c13` , sum(c16) AS `c16` , sum(c11) AS `c11` , sum(c10) AS `c10` , sum(c14) AS `c14` , sum(c9) AS `c9` , sum(c12) AS `c12` , sum(c17) AS `c17` , sum(c22)c22 , sum(c39)c39 , sum(c40)c40
+            FROM slow_fold_constant_case_when
+            WHERE slow_fold_constant_case_when.`c43` IN ('______')
+                    AND (( ((`c72` IN ('______'))
+                    OR (`c73` IN ('______'))))
+                    OR (`c63` IN ('______')))
+            GROUP BY  _______0 limit 3000 ) t ) t lateral view explode ( `_______temp` ) c0 AS _____
+            WHERE _____ is NOT null
+                    AND _____ != '______' ) t
+            GROUP BY  grouping sets((_____) ,())
+            ORDER BY  _____ nulls last limit 3000 ) t
+        ORDER BY  _____ nulls last
+            """
+
+    sql "set enable_fold_constant_by_be=false"
+    test {
+        sql sqlStr
+        time 60000
```

Comment:
is it acceptable in P0 rather than P1?

Reply:
It should be fast, so we don't need to move to p1